src/psalm.php

```diff
@@ -127,7 +127,7 @@
     $options['r'] = $options['root'];
 }
 
-$current_dir = (string)getcwd() . DIRECTORY_SEPARATOR;
+$current_dir = (string)getcwd();
```

Comment:
I'm not quite sure why we have the trailing `/`. This was making fail the find autoloader since we ended up with `//` in the path